### PR TITLE
fix typo in wallet.js error handling

### PR DESCRIPTION
**Pull request by RNHTTR**
fix `indexOf(``ccount...` with `indexOf(``account...`

### CODE REVIEW

File: packages/frontend/src/utils/wallet.js [hunk 1]
```diff
@@ -188,7 +188,7 @@ class Wallet {
         } catch (error) {
             console.log('Error loading account:', error.message);
 
-            if (error.toString().indexOf(`ccount ${this.accountId} does not exist while viewing`) !== -1) {
+            if (error.toString().indexOf(`account ${this.accountId} does not exist while viewing`) !== -1) {
```
Comment (MaximusHaximus) [inline packages/frontend/src/utils/wallet.js:191]: I think we had issues with the first character not always being capitalized or lowercase across API versions, so we actually started matching just `does not exist while viewing` instead  (see: https://github.com/near/near-wallet/blob/36ad3602820b6a27f739f4504dd1a17589e133e0/packages/frontend/src/components/send/components/InputAccountId.js#L174) -- which has the other benefit of being more obvious that we _intentionally_ omitted the beginning of the string :). I'm happy to merge a PR that changes it to `does not exist while viewing` if you'd like to do that instead :)

**Review by MaximusHaximus — CHANGES_REQUESTED** [review]
Actually, leaving off the first character here was intentional, but not obviously so.  Added a comment in-line :)

File: packages/frontend/src/utils/wallet.js (hunk 1 above)
Comment (RNHTTR) [reply packages/frontend/src/utils/wallet.js]: Aha! that makes sense. I updated the PR :)

Comment (MaximusHaximus) [reply packages/frontend/src/utils/wallet.js]: <3 :) thanks so much!

**Review by MaximusHaximus — APPROVED** [review]
<3 consistency++ :)

**Pull request merged**